Error :(-215 Assertion failed)

The code crash with this error  :

line 222, in predict
    outs = net.forward(getOutputLayers(net))
cv2.error: OpenCV(4.7.0) /io/opencv/modules/dnn/src/layers/fast_convolution/winograd_3x3s1_f63.cpp:147: error: (-215:Assertion failed) _FX_WINO_IBLOCK == 3 && _FX_WINO_KBLOCK == 4 in function '_fx_winograd_accum_f32'

double free or corruption (out)
Aborted

Can't find where is the issue, it's been like 2 years since i used opencv with c++ and even then i wasn't that good.  I hope you can help me out .

--- a/setup.py
+++ b/setup.py
@@ -21,6 +21,6 @@ setup(
         'Operating System :: OS Independent',
     ],
     packages=find_packages(),
-    install_requires = ['opencv-python', 'Pillow', 'numpy', 'requests', 'selenium==3.141.0'],
+    install_requires = ['opencv-python==4.6.0.66', 'Pillow', 'numpy', 'requests', 'selenium==3.141.0'],
     python_requires='>=3.6',
 )

